src/package-tree.ts

```diff
@@ -194,3 +236,65 @@ function main() {
   // const packageTreeWithPOI = populatePOIInPackageTree(packageTreeWithPath);
   throw new Error('not implemented');
 }
+
+/**
+ * Takes in a list of depndencies and returns a constructed PackageTree
+ * @param packageLockJson the package-lock.json file of the root project
+ */
+export async function getPackageTreeFromDependencyList(
+    dependencies: {},
+    packageLockJson: {}): Promise<Array<PackageTree<undefined>>> {
+  // Step 1: For each dependency create a PackageTree obj with the name and
+  // version fields populated
+  if (!dependencies) {
+    return [];
+  }
+
+  const dependencyArr: Array<[string, string]> = Object.entries(dependencies);
```

Comment:
Did you end up changing `Object.entries`?

Reply:
Yes, I added it in tsconfig.json